src/libs/ui/src/molecules/SnapShotRating/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { useReviewContext } from '../ReviewUseContext';
+import ReviewBar from '../ReviewBar';
+
+interface ReviewMoleculeProps {
+  className?: string;
+  reviews: { stars: number; count: number }[];
+}
+
+const SnapShotRating: React.FC<ReviewMoleculeProps> = ({ className, reviews }) => {
+  const { updateBarRatingFilter, setSelectedRatings } = useReviewContext();
+  // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+  const [, setSelectedRatingsLocal] = useState<string[]>([]);
```

Comment:
what is use of this useState, if not used please remove it, and make sure all data and filter comes from useContext